src/sparselm/model/_adaptive_lasso.py

```diff
@@ -380,22 +486,22 @@ class AdaptiveSparseGroupLasso(AdaptiveLasso, SparseGroupLasso):
 
     def __init__(
         self,
-        groups,
-        l1_ratio=0.5,
```

Comment:
Why should we go back to using a "ratio"?

Reply:
I never changed this for Lasso-based regressors. I suggest keeping it like that since in this case the regularization is a convex combination and that is how `Lasso` in scikit-learn is implemented.

Reply:
Okay you can keep this in adaptive Lasso, but maybe never change to a ratio in MIQP_L0?